src/Sentry/Extensibility/SentryStackTraceFactory.cs

```diff
@@ -68,47 +69,41 @@ internal SentryStackTrace Create(StackTrace stackTrace, bool isCurrentStackTrace
                 stacktrace.Frames.Add(frame);
             }
 
-            return stacktrace.Frames.Count == 0
-                ? null
-                : stacktrace;
+            return stacktrace.Frames.Count != 0
```

Comment:
Just curious: Why flip this?

Reply:
I personally prefer when the positive/dominant branch appears first and the fallback appears last. I saw you had it like this in some other places anyway, so I flipped it to make it more consistent.

Reply:
I'm happy with that. I have no strong preferences here and do appreciate consistency so thanks for improcing this. Generally mixing different changes in the same PR make it harder to review. This is less important in small PRs but if they are huge, it's hard enough to go through them for what they are for